priv/repo/migrations/20170505224222_add_github_repo_and_owner_to_project.exs

```diff
@@ -0,0 +1,10 @@
+defmodule CodeCorps.Repo.Migrations.AddGithubRepoAndOwnerToProject do
+  use Ecto.Migration
+
+  def change do
+    alter table(:projects) do
+      add :github_repo, :string
+      add :github_owner, :string
```

Comment:
I think this means we will no longer require a `github_id`. Same would go with ember's side. Would you agree?